src/coreclr/build-runtime.cmd

```diff
@@ -152,6 +154,7 @@ if /i "%1" == "-enforcepgo"          (set __EnforcePgo=1&shift&goto Arg_Loop)
 if /i "%1" == "-pgodatapath"         (set __PgoOptDataPath=%~2&set __PgoOptimize=1&shift&shift&goto Arg_Loop)
 if /i "%1" == "-component"           (set __RequestedBuildComponents=%__RequestedBuildComponents%-%2&set "__remainingArgs=!__remainingArgs:*%2=!"&shift&shift&goto Arg_Loop)
 if /i "%1" == "-fsanitize"           (set __CMakeArgs=%__CMakeArgs% "-DCLR_CMAKE_ENABLE_SANITIZERS=%2"&shift&shift&goto Arg_Loop)
+if /i "%1" == "-keepnativesymbols"   (set __CMakeArgs=%__CMakeArgs% "-DCLR_CMAKE_KEEP_NATIVE_SYMBOLS=true"&shift&goto Arg_Loop)
```

Comment:
I'd rather use https://github.com/dotnet/runtime/pull/114466 for that instead.

Reply:
This way it is useful when you run the build-runtime scripts directly, to have faster roundtrip of runtime build.

Reply:
I don't think the batch gods are happy as the build fails locally for me with the ever clear `The system cannot find the path specified.`. The command line processing in this batch file can be brittle.

Reply:
How do you call it? It worked fine for me.

I will enable windows build on CI to see if it works there.

Reply:
`.\build.cmd -Subset clr+libs -os browser -arch wasm`